docs/api-docs/analytics/bodl-overview.md

```diff
@@ -0,0 +1,407 @@
+
+```handlebars title="Sample Script Code Start: Purchase Event" lineNumbers
+<script>
+  fetch('/api/storefront/order/{{checkout.order.id}}', {
+    credentials: 'same-origin'
+  })
+  .then(function(response) {
+    return response.json();
+  })
+  .then(function(orderJson) {
+    var orderQty = 0;
+    var lineItems = [];
+
+    for (i = 0; i < orderJson.lineItems.physicalItems.length; i++) {
+      var thisItem = orderJson.lineItems.physicalItems[i];
+      orderQty += thisItem.quantity;
+      lineItems.push({
+        "content_id": thisItem.productId,
+        "content_name": thisItem.name,
+        "currency": orderJson.currency.code,
+        "price": thisItem.salePrice,
+        "value": thisItem.extendedSalePrice,
+        "quantity": thisItem.quantity,
+        "content_type": "product_group"
+      });
+    }
+
+    for (i = 0; i < orderJson.lineItems.digitalItems.length; i++) {
+      var thisItem = orderJson.lineItems.digitalItems[i];
+      orderQty += thisItem.quantity;
+      lineItems.push({
+        "content_id": thisItem.productId,
+        "content_name": thisItem.name,
+        "currency": orderJson.currency.code,
+        "price": thisItem.salePrice,
+        "value": thisItem.extendedSalePrice,
+        "quantity": thisItem.quantity,
+        "content_type": "product_group"
+      });
+    }
+
+    for (i = 0; i < orderJson.lineItems.giftCertificates.length; i++) {
+      var thisItem = orderJson.lineItems.giftCertificates[i];
+      orderQty += thisItem.quantity;
+      lineItems.push({
+        "content_id": thisItem.type,
+        "content_name": thisItem.name,
+        "currency": orderJson.currency.code,
+        "price": thisItem.amount,
+        "value": thisItem.amount,
+        "quantity": thisItem.quantity,
+        "content_type": "product_group"
+      });
+    }
+
+    sampleAnalyticsProvider.instance('<%= property_id %>').track('Purchase', {
+        "contents": lineItems,
+        "value": orderJson.orderAmount,
+        "quantity": orderQty,
+        "currency": orderJson.currency.code
+    });
+  });
+</script>
+```
+
+### Registration
+The following snippet executes when a shopper successfully creates a new account. It initiates capture of any data the sample analytics provider collects about account registration.
+
+```handlebars title="Sample Script Code Start: Registration" lineNumbers
+<script>
+  if (window.location.pathname.indexOf('/login.php') === 0 && BODL.getQueryParamValue('action') === 'account_created') {
+    sampleAnalyticsProvider.instance('<%= property_id %>').track('Registration');
+  }
+</script>
+```
+
+### Search
+This snippet tracks when an end-user searches for products. Please take note that there is a built-in tracker for the category. However, we have commented out the tracker due to known distortions, which cause reporting of only the first category ID. Only reactivate if needed; use it at your own risk.
```

Comment:
@traci707 since we've removed the search snippet, let's either change the description to reflect the lack of snippet, or cut the section altogether.